cheatsheets/Cross-Site_Request_Forgery_Prevention_Cheat_Sheet.md

```diff
@@ -254,39 +250,6 @@ While all the techniques referenced here do not require any user interaction, so
 
 While these are a very strong CSRF defense, it does create a huge impact on the user experience. For applications that are in need of high security for some operations (password change, money transfer etc.), these techniques should be used along with token based mitigation. Please note that tokens by themselves can mitigate CSRF, developers should use these techniques only to achieve additional security for their high sensitive operations.
 
-# Not So Popular CSRF Mitigations
```

Comment:
I  personally know lot of developers who think content-type header validation can be used as a CSRF defense and know few folks who think triple submit cookie is better than double submit cookie.

My understanding of cheat sheet is that when people come to it, they need to know whats right way to do and whats wrong way. We dont need to highlight all possible "wrong" ways but at least mention the most widely known one's. Hence, I recommend adding these things back. We can add it with different title/ strip them to a line/two instead of how they are expanded now. What do you guys think?

Reply:
I agree, it would be awesome to leave them but in shorter from!

Reply:
Yup, fine with me.